### PR TITLE
New: added the option --xml-output-format for the command objj

### PR DESCRIPTION
**Pull request by Dogild**
Previously, it was not possible to specify the desired output format of the command objj.
Now we can have either the default format or a xml format if --xml-output-format is added.

### CODE REVIEW

**Comment by Dogild** [comment]
@mrcarlberg, @primalmotion any objections for this PR ?

File: Objective-J/CommonJS/lib/objective-j.js [hunk 1]
```diff
@@ -94,6 +94,7 @@ exports.run = function(args)
             print("  -I, --objj-include-paths       include a specific framework paths")
             print("  -h, --help                     print this help");
             print("  -m, --multifiles               launch objj on several files")
+            print("  --xml-output-format            specify the output format in xml.")
```
Comment (primalmotion) [inline Objective-J/CommonJS/lib/objective-j.js]: can you add -x so all parameters have their short version? also I think the name should simply be --xml

File: Objective-J/ObjJAcornCompiler.js [hunk 2]
```diff
@@ -442,14 +457,32 @@ ObjJAcornCompiler.prototype.compilePass2 = function()
     compile(this.tokens, new Scope(null ,{ compiler: this }), pass2);
     for (var i = 0; i < this.warnings.length; i++)
     {
-       var message = this.prettifyMessage(this.warnings[i], "WARNING");
+        var warning = this.warnings[i],
+            type = "WARNING";
+
+        var message = this.prettifyMessage(warning, type);
```
Comment (primalmotion) [inline Objective-J/ObjJAcornCompiler.js]: do you need to prettify the message when xml output is set to xml?

**Comment by primalmotion** [comment]
I've added a few comments, otherwise, looks great!

File: Objective-J/ObjJAcornCompiler.js (hunk 2 above)
Comment (Dogild) [inline Objective-J/ObjJAcornCompiler.js]: Yes, I use the message in the xml

**Pull request merged**